src/Symfony/Component/HttpFoundation/Tests/Test/Constraint/ResponseIsRedirectedTest.php

```diff
@@ -29,9 +28,8 @@ public function testConstraint()
         try {
             $constraint->evaluate(new Response('Body content'));
         } catch (ExpectationFailedException $e) {
-            $exceptionMessage = TestFailure::exceptionToString($e);
-            $this->assertStringContainsString("Failed asserting that the Response is redirected.\nHTTP/1.0 200 OK", $exceptionMessage);
-            $this->assertStringContainsString('Body content', $exceptionMessage);
+            $this->assertStringContainsString("Failed asserting that the Response is redirected.\nHTTP/1.0 200 OK", $e->getMessage());
+            $this->assertStringContainsString('Body content', $e->getMessage());
```

Comment:
Can't we work with `expectException()` here too?

Reply:
yes indeed, I updated some more tests accordingly